tools/migrator/src/index.ts

```diff
@@ -74,8 +81,8 @@ program
     }
 
     if (res.error) {
-      logger.warn("Error while running migrations:")
-      logger.warn(JSON.stringify(res.error))
+      logger.warn("Error while running migrations")
+      console.dir(res.error, { depth: null }) // prints stack trace
```

Comment:
`logger.warn(JSON.stringify(res.error))` does not print stack trace and is super annoying when getting errors